lte/gateway/configs/pipelined.yml_prod

```diff
@@ -167,12 +171,20 @@ he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
 
 ovs_gtp_stats_polling_interval: 180
 
+# Make as a True when gnb ip will support for uplink
+ovs_multi_tunnel: False
```

Comment:
Can you please check if without  'ovs_multi_tunnel' things are working. Is it a must needed.

Reply:
This flag is used whenever uplink flow is added in ovs.